rasa/server.py

```diff
@@ -500,12 +500,14 @@ async def append_events(request: Request, conversation_id: Text):
             async with app.agent.lock_store.lock(conversation_id):
                 processor = app.agent.create_processor()
                 tracker = processor.get_tracker(conversation_id)
+                output_channel = _get_output_channel(request, tracker)
                 _validate_tracker(tracker, conversation_id)
 
                 events = _get_events_from_request_body(request)
 
                 for event in events:
                     tracker.update(event, app.agent.domain)
+                await processor._send_bot_messages(events, tracker, output_channel)
```

Comment:
Let's change `MessageProcessor._send_bot_messages` to `MessageProcessor.send_bot_messages` (and update all usages).

Reply:
Good idea @federicotdn. Will submit revision with processor methods `_send_bot_messages`, `_schedule_reminders` and `_cancel_reminders` bundled in a public method named `execute_side_effects` or something, and add a test.